src/main/java/org/opentripplanner/graph_builder/module/GtfsFeedId.java

```diff
@@ -16,26 +18,40 @@
      */
     private static int FEED_ID_COUNTER = 1;
 
+    /**
+     * A Map that keeps track of which feed ids were generated to make sure there are no duplicates
+     */
+    private static final Set<String> seenIds = new HashSet<>();
```

Comment:
Suggestion: perhaps rename to `knownIds`.

Reply:
I prefer `seenIds`.

Reply:
Not a map, is it?